### ISSUE TITLE
Show comment block parameters in relevant sections

### ISSUE THREAD

**Issue by JasonTheAdams** [piklist/Piklist-Documentation#4]
It would be really handy to have the comment blocks also display the comment block parameters on the metabox, settings, term, etc. pages, or at least have a link to them.

I forget there's a page specifically for the comment block paramters, and go to the Meta-Boxes page expecting to see them there.

**Comment by sbruner** [comment]
Agreed. We discussed using taxonomies for this. Feel free to set them up.  I'll look into them as well later this week
